<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransportWidget</class>
 <widget class="QWidget" name="TransportWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3"/>
     </item>
     <item>
      <widget class="QLabel" name="coverLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Cover</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="songNameLabel">
         <property name="text">
          <string>Heavy is the Crown</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="artistNameLabel">
         <property name="text">
          <string>Linkin Park</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="shuffleButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../resources/shuffle.svg</normaloff>../resources/shuffle.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="prevButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../resources/skip-back.svg</normaloff>../resources/skip-back.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="playButton">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../resources/play.svg</normaloff>../resources/play.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="nextButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../resources/skip-forward.svg</normaloff>../resources/skip-forward.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="loopButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../resources/repeat.svg</normaloff>../resources/repeat.svg</iconset>
           </property>
           <property name="autoRepeat">
            <bool>false</bool>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="currentTimeLabel">
           <property name="text">
            <string>0:00</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="seekSlider">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="durationLabel">
           <property name="text">
            <string>3:37</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
